discord_thread_summarizer_using_openai/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "ai_samples"
+name = "discord_thread_summarizer_using_openai"
+version = "0.1.0"
+distribution = "2201.5.0"
```

Comment:
Can't we use U6?

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/59aea9bda909054c8bca4433c79691e50b02e0cb